src/main/resources/io/federecio/dropwizard/swagger/index.ftl

```diff
@@ -3,24 +3,24 @@
 <html>
 <head>
   <title>Swagger UI</title>
-  <link href='${swaggerAssetsPath}/css/typography.css' media='screen' rel='stylesheet' type='text/css'/>
-  <link href='${swaggerAssetsPath}/css/reset.css' media='screen' rel='stylesheet' type='text/css'/>
-  <link href='${swaggerAssetsPath}/css/screen.css' media='screen' rel='stylesheet' type='text/css'/>
-  <link href='${swaggerAssetsPath}/css/reset.css' media='print' rel='stylesheet' type='text/css'/>
-  <link href='${swaggerAssetsPath}/css/screen.css' media='print' rel='stylesheet' type='text/css'/>
-  <script src='${swaggerAssetsPath}/lib/jquery-1.8.0.min.js' type='text/javascript'></script>
-  <script src='${swaggerAssetsPath}/lib/jquery.slideto.min.js' type='text/javascript'></script>
-  <script src='${swaggerAssetsPath}/lib/jquery.wiggle.min.js' type='text/javascript'></script>
-  <script src='${swaggerAssetsPath}/lib/jquery.ba-bbq.min.js' type='text/javascript'></script>
-  <script src='${swaggerAssetsPath}/lib/handlebars-2.0.0.js' type='text/javascript'></script>
-  <script src='${swaggerAssetsPath}/lib/underscore-min.js' type='text/javascript'></script>
-  <script src='${swaggerAssetsPath}/lib/backbone-min.js' type='text/javascript'></script>
-  <script src='${swaggerAssetsPath}/swagger-ui.js' type='text/javascript'></script>
-  <script src='${swaggerAssetsPath}/lib/highlight.7.3.pack.js' type='text/javascript'></script>
-  <script src='${swaggerAssetsPath}/lib/marked.js' type='text/javascript'></script>
+  <link href='..${swaggerAssetsPath}/css/typography.css' media='screen' rel='stylesheet' type='text/css'/>
```

Comment:
Shouldn't it be `../` just to be safe when `${swaggerAssetsPath}` does not contain a leading `/`?